test/ex_doc/formatter/html/templates_test.exs

```diff
@@ -166,14 +170,14 @@ defmodule ExDoc.Formatter.HTML.TemplatesTest do
   ## LISTING
 
   test "site title text links to homepage_url when set" do
-    content = Templates.sidebar_template(doc_config(), @empty_nodes_map)
+    content = doc_config() |> Templates.sidebar_template(@empty_nodes_map) |> normalize_eol()
```

Comment:
We don't need this. We should just remove `\n` from the regexes below because `\s` matches any space and we don't actually care in HTML if it is a new line or a regular space:

    "a\r\nb" =~ ~r/a\s*b/
    true